Apps/phsignalfx/__init__.py

```diff
@@ -0,0 +1,4 @@
+# File: __init__.py
+# Copyright (c) 2016-2021 Splunk Inc.
```

Comment:
Please update the license year to 2021 only as it's an initial release. Do the changes for all the .py and HTML files.

Reply:
Fixed